lib/rubocop/cop/rspec/factory_bot/syntax_methods.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use the shorthands from `FactoryBot::Syntax::Methods` in your specs.
+        #
+        # The auto-correction is marked as unsafe because the cop
```

Comment:
There's a recently added `@safety` tag. I believe it adds a comment to the docs in a specific way. Example https://github.com/rubocop/rubocop/blob/783cb69643ab9a40fee9e8917aa557299be5b59a/lib/rubocop/cop/lint/number_conversion.rb#L21

Reply:
Had to remove it because `rake default` said `@safety` is not known.

Reply:
Wondering how it works 🤔 
What if you `bundle update rubocop`?

Reply:
I added the `.yardopts` file from `rubocop-performance` and now it works.